src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.*;
+import java.io.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final String FILE_PATH = "./data/duke.txt";
+    public static void main(String[] args) throws Exception {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm Yappy\n    What can I do for you?");
+        System.out.println("    ____________________________________________________________");
+
+        ArrayList<Task> AL = new ArrayList<>();
+
+        loadTasksFromFile(AL, FILE_PATH);
+
+        while(true) {
+            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
Just a small note: I feel that you could adhere to the use of 'bufferedReader' instead of 'br' because later on in the code 'bufferedWriter' was used to name the BufferedWriter instance; just to keep it consistent